docs/maps/maps-getting-started.asciidoc

```diff
@@ -177,10 +177,16 @@ image::maps/images/gs_dashboard_with_map.png[]
 
 .. Open a tooltip by clicking anywhere in the United States vector.
 
-.. To show only documents where **geo.src** is **US**, click the filter icon in the row for **ISO 3066-1 alpha-2**.
+.. To show only documents where **geo.src** is **US**, click the filter icon image:maps/images/gs-filter-icon.png[filter icon, width=24px]in the row for **ISO 3066-1 alpha-2**.
 +
 [role="screenshot"]
-image::maps/images/gs_dashboard_with_terms_filter.png[]
+image::maps/images/gs_tooltip_filter.png[Tooltip on map, width=75%]
+
+
+Your filtered map should look similar to this:
+
+[role="screenshot"]
+image::maps/images/gs_dashboard_with_map.png[Map showing filtered data]
```

Comment:
This should use a different image then gs_dashboard_with_map.png so users can see the difference of applying the filter.